.eslintrc-react.js

```diff
@@ -0,0 +1,20 @@
+
+module.exports = {
+    "extends": [
+        "./.eslintrc.js",
+        "airbnb",
+        "plugin:react/recommended"
```

Comment:
what's the difference between this and it being in the plugin array?

Reply:
My read based on [this section in the docs](https://eslint.org/docs/user-guide/configuring#using-the-configuration-from-a-plugin) is this syntax is actually getting the `recommended` ruleset from the react plugin, not configuring anything particular about the plugin.